Add 4 new questions to GitHub Foundations around GitHub administration based on Microsoft Learn guides and GitHub official documentation. (#209)
FidelusAleksander/ghcertified: content/questions/foundations/question-109.md

---
archetype: "questions"
title: "Question 109"

> question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub? (Choose four.)"

Reviewer: After adding TOTP

```suggestion
question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub? (Choose five.)"
```

draft: false
---

> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
- [x] Passkey
- [x] Text message
- [ ] Phone call
> While text messages are supported in some countries, phone calls are not
- [ ] Email
- [x] Security key
- [x] GitHub mobile

> - [ ] Time-based one-time password (TOTP)

Reviewer: Isn't this correct?

```suggestion
- [x] Time-based one-time password (TOTP)
```

Reviewer 2: That is on me. I didn't make it clear that TOTP should be included AND marked as correct.

---
archetype: "questions"
title: "Question 109"

> question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub?"

Reviewer: ```suggestion
question: "What are the different forms of two-factor or multi-factor authentication supported by GitHub? (Choose four.)"
```

draft: false
---

> https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication
- [x] Passkey
- [x] Text message
- [ ] Phone call

> > While text messages are supported in some countries, there is no mention of phone calls for 2FA/MFA.

Reviewer: ```suggestion
> While text messages are supported in some countries, phone calls are not
```

- [ ] Email

> > There is no mention of email as an option to recieve a code for MFA/2FA.

Reviewer 2: ```suggestion
> There is no mention of email as an option to receive a code for MFA/2FA.
```

> > There is no mention of email as an option to receive a code for MFA/2FA.

Reviewer: Don't assume that people actually will click and read the links in the light bulb - that would be cheating on the actual exam. Therefore explanations shouldn't include "there is no mention in .."
```suggestion
```

- [x] Security key
- [x] GitHub mobile

> - [ ] Biometrics

Reviewer 2: @JBonifacio Personally, I would remove it, because while GitHub does not directly use Biometrics, the passkeys (in the browser), it gets used high level with Security keys:

```
Select the appropriate option in the prompt. Depending on your security key configuration, you may type a PIN, complete a biometric prompt, or use a physical security key.
```

Also, TOTP is supported, but not included in the prompts here. It may be a good idea to add that.
